ansible/roles/software/ckan/catalog/ckan-app/files/etc/supervisor/conf.d/supervisord_web_mem_cpu.conf

```diff
@@ -0,0 +1,7 @@
+; supervisorctl config file to restart apache when mem/cpu resource is critically low
+[program:cpu-mem-check]
+command=/usr/bin/cpu-mem-check.sh
+stdout_logfile=/var/log/ckan/cpu-mem-check.log
+redirect_stderr=true
+autostart={{ datagov_in_service | default(true) | ternary('true', 'false') }}
+autorestart={{ datagov_in_service | default(true) | ternary('true', 'false') }}
```

Comment:
Let's create a new Ansible variable to enable/disable this. I think I'd like to run it on a single host to start and it would be nice to flip that on/off by Ansible configuration.